packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +40,30 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        const commandsUsed = () => {
+            return process.argv.filter((val) => commands.find(({ name }) => name === val));
+        };
+
+        if (externalPkg && commandsUsed().length === 1) {
+            try {
+                if (commandsUsed().includes(externalPkg.name)) {
+                    const { name, version } = require(`@webpack-cli/${externalPkg.name}/package.json`);
+                    process.stdout.write(`\n${name} ${version}`);
+                } else {
+                    const { name, version } = require(`${externalPkg.name}/package.json`);
+                    process.stdout.write(`\n${name} ${version}`);
```

Comment:
Possible scope for refactoring 🤔

Reply:
It can be refactored like below - 
```
const { name, version } = commandsUsed().includes(externalPkg.name) ? require(`@webpack-cli/${externalPkg.name}/package.json`) : require(`${externalPkg.name}/package.json`);
process.stdout.write(`\n${name} ${version}`);
```

Reply:
^ is not readable at all, won't suggest